client/lib/plugins/wporg-data/actions.js

```diff
@@ -77,8 +92,26 @@ var PluginsDataActions = {
 		} );
 	}, 25 ),
 
+	fetchCuratedList: function( category, slugs ) {
+		const data = CuratedPlugins.filter( function( plugin ) {
+			return includes( slugs, plugin.slug );
+		} );
+		debug( 'curated plugin list', category, slugs );
+		_fetchingLists[ category ] = null;
+		_lastFetchedPagePerCategory[ category ] = 1;
+		_totalPagesPerCategory[ category ] = 1;
+		Dispatcher.handleServerAction( {
+			type: 'RECEIVE_WPORG_PLUGINS_LIST',
+			action: 'FETCH_WPORG_PLUGINS_LIST',
+			page: 1,
+			category: category,
+			data: utils.normalizePluginsList( data ),
+			error: null
+		} );
```

Comment:
This way we can show something on the initial load, but have correct data eventually populate the plugin, like rating. (Which I believe we need anyway for the plugin specific view).